### PR TITLE
SIDM-3816 idam-aat2 configuration

### PR DESCRIPTION
**Pull request by aidanfitzpatrickamido**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SIDM-3816

### Change description ###

increasing version number to 0.2.3

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: k8s/aat/common/idam/idam-api.yaml [hunk 1]
```diff
@@ -15,13 +15,21 @@ spec:
...
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: idam-api
-    version: 0.2.2
+    version: 0.2.3
   values:
     java:
-      image: hmctspublic.azurecr.io/idam/api:aat-16d37681
-      ingressHost: idam-api.service.core-compute-aat.internal
+      image: hmctspublic.azurecr.io/idam/api:prod
+      imagePullPolicy: Always
```
Comment (henrydobson) [inline k8s/aat/common/idam/idam-api.yaml]: Not required. Remove @aidanfitzpatrickamido

File: k8s/aat/common/idam/idam-web-public.yaml [hunk 2]
```diff
@@ -15,13 +15,15 @@ spec:
...
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: idam-web-public
-    version: 0.2.2
+    version: 0.2.3
   values:
     java:
-      image: hmctspublic.azurecr.io/idam/web-public:aat-1ebdebd1
-      ingressHost: idam-web-public.service.core-compute-aat.internal
+      image: hmctspublic.azurecr.io/idam/web-public:prod
+      ingressHost: idam-web-public.aat.platform.hmcts.net
```
Comment (henrydobson) [inline k8s/aat/common/idam/idam-web-public.yaml]:
```suggestion
      ingressHost: idam-web-public-aat2.aat.platform.hmcts.net
```

File: CODEOWNERS [hunk 3]
```diff
@@ -178,6 +178,9 @@ k8s/**/common/bar/identity.yaml @hmcts/Fees
 k8s/**/common/bar/namespace.yaml @hmcts/Fees
 
 # IDAM
+k8s/**/common/idam/idam-api.yaml @hmcts/idam
+k8s/**/common/idam/idam-web-public.yaml @hmcts/idam
+k8s/**/common/idam/idam-web-admin.yaml @hmcts/idam
```
Comment (adusumillipraveen) [inline CODEOWNERS]: Let it be aat for now as prod would need approval from secops in terms of CVEs.

```suggestion
k8s/aat/common/idam/idam-api.yaml @hmcts/idam
k8s/aat/common/idam/idam-web-public.yaml @hmcts/idam
k8s/aat/common/idam/idam-web-admin.yaml @hmcts/idam
```

File: k8s/aat/common/idam/idam-api.yaml [hunk 4]
```diff
@@ -15,13 +15,20 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: idam-api
-    version: 0.2.2
+    version: 0.2.3
   values:
     java:
-      image: hmctspublic.azurecr.io/idam/api:aat-16d37681
-      ingressHost: idam-api.service.core-compute-aat.internal
+      image: hmctspublic.azurecr.io/idam/api:prod
```
Comment (timja) [inline k8s/aat/common/idam/idam-api.yaml]: is prod tag a thing? normally we just put latest in for the first merge, and then flux will automatically update it
```suggestion
      image: hmctspublic.azurecr.io/idam/api:latest
```

File: k8s/aat/common/idam/idam-api.yaml [hunk 5]
```diff
@@ -15,13 +15,20 @@ spec:
...
+      replicas: 1
```
Comment (timja) [inline k8s/aat/common/idam/idam-api.yaml]: why only 1 replica?

Comment (henrydobson) [reply k8s/aat/common/idam/idam-api.yaml]: We have a problem to solve of pod level session affinity. Until then, 1 replica.

File: k8s/aat/common/idam/idam-web-public.yaml [hunk 6]
```diff
@@ -15,13 +15,15 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: idam-web-public
-    version: 0.2.2
+    version: 0.2.3
   values:
     java:
-      image: hmctspublic.azurecr.io/idam/web-public:aat-1ebdebd1
-      ingressHost: idam-web-public.service.core-compute-aat.internal
+      image: hmctspublic.azurecr.io/idam/web-public:prod
```
Comment (timja) [inline k8s/aat/common/idam/idam-web-public.yaml]:
```suggestion
      image: hmctspublic.azurecr.io/idam/web-public:latest
```

File: k8s/aat/common/idam/idam-api.yaml (hunk 5 above)
Comment (timja) [reply k8s/aat/common/idam/idam-api.yaml]: and it's not a problem with multiple clusters? as even though you're setting 1 here there will be another cluster running with it

File: k8s/aat/common/idam/idam-web-admin.yaml [hunk 7]
```diff
@@ -15,13 +15,16 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: idam-web-admin
-    version: 0.2.2
+    version: 0.2.3
   values:
     java:
-      image: hmctspublic.azurecr.io/idam/web-admin:aat-3f69a75f
-      ingressHost: idam-web-admin.service.core-compute-aat.internal
+      image: hmctspublic.azurecr.io/idam/web-admin:prod
```
Comment (henrydobson) [inline k8s/aat/common/idam/idam-web-admin.yaml]:
```suggestion
      image: hmctspublic.azurecr.io/idam/web-admin:latest
```

**Pull request merged**